packages/tasks/src/local-apps.ts

```diff
@@ -180,6 +183,33 @@ const snippetVllm = (model: ModelData): LocalAppSnippet[] => {
 		},
 	];
 };
+const snippetTgi = (model: ModelData): LocalAppSnippet[] => {
+	const runCommand = [
+		"# Call the server using curl:",
+		`curl -X POST "http://localhost:8000/v1/chat/completions" \\`,
+		`	-H "Content-Type: application/json" \\`,
+		`	--data '{`,
+		`		"model": "${model.id}",`,
+		`		"messages": [`,
+		`			{"role": "user", "content": "What is the capital of France?"}`,
```

Comment:
This assumes that all models are instruct, we would need to update this once Mishig's PR is merged: https://github.com/huggingface/huggingface.js/pull/954

Or happy to merge this now @mishig25 - in case you want to take care of this in your PR - whatever you think is the best.